Source/WebCore/svg/SVGComponentTransferFunctionElement.h

```diff
@@ -53,15 +53,15 @@ struct SVGPropertyTraits<ComponentTransferType> {
 
     static ComponentTransferType fromString(const String& value)
     {
-        if (value == "identity")
+        if (value == "identity"_s)
```

Comment:
Yet another example where `SortedArrayMap` is the best idiom, but no need to try to fix all of these.

Reply:
Will update.